### DISCUSSION TITLE
Make it easier to see file additions/modifications and registry edits from a sandboxed program

### DISCUSSION THREAD

**Discussion by Koakuma-spec** [sandboxie-plus/Sandboxie#2766] [Various ideas and suggestions]
I want to be able to test untrusted programs with Sandboxie while being able to see what kinds of files and registry entries it modifies. BSA (Buster Sandbox Analyzer) is said to accomplish this, but I have trouble getting this started after downloading as it complains about missing .dlls that aren't actually missing. After reading a thread by the maintainer on Wilders Security Forums, it seems that BSA isn't always compatible with Sandboxie depending on what version the end-user is using (I'm on Sandboxie-Plus v1.7.2)

Instead of messing with a third-party software that's not necessarily compatible with my Sandboxie version, why not build in this kind of feature for easier analysis?

**Comment by DavidXanatos** [comment]
there kind of is, its the resource access monitor, what are you mising htere?

**Reply by Koakuma-spec** [reply]
Where is that?

**Reply by isaak654** [reply]
I think [this](https://user-images.githubusercontent.com/12372772/227929730-eba80e05-c654-440c-b23c-b08f2d142e23.png) is what he meant.
But there is a downside... although it allows you to selectively filter only registry keys and files and export the output as .log file, it does not perform any kind of comparison.

**Comment by bastik-1001** [comment]
For viewing registry changes, there are the [SandboxToys2](https://github.com/blap/SandboxToys2).

**Reply by isaak654** [reply]
In addition, there is an open feature request: #60

**Reply by Koakuma-spec** [reply]
The links to the .exe file in the forum appear to be outdated. One doesn't work, for the other one my browser says it "can't be downloaded securely".

**Reply by isaak654** [reply]
I don't know who you were trying to reply, but I mentioned the feature request #60 for informational purposes only.

If you want to report something about [SandboxToys2](https://github.com/blap/SandboxToys2), you can open a new issue there.
